package.json

```
@@ -1,8 +1,8 @@
 {
   "name": "svelte-state-renderer",
-  "version": "3.3.0",
+  "version": "4.0.0",
   "description": "abstract-state-router renderer for Svelte",
-  "main": "index.js",
+  "main": "index.svelte.js",
```

```suggestion
  "main": "index.svelte.js",
  "type": "module",
```
Now that the main file is ESM, I think we should set `type` in the package.json